src/status_im2/contexts/communities/requests/actions/view.cljs

```diff
@@ -81,15 +88,26 @@
 
 (defn community-rules-list
   [rules]
-  [list/flat-list
+  [rn/flat-list
    {:shows-horizontal-scroll-indicator false
     :data                              rules
     :separator                         [rn/view {:margin-top 1}]
     :render-fn                         community-rule-item}])
 
+(def request-cooldown-ms (* 24 60 60 1000))
```

Comment:
this probably should be somewhere in constants or in setup

Reply:
yeah it makes sense to have it somewhere broader in scope